aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkSettings.kt

```diff
@@ -29,6 +29,11 @@ class SdkSettings private constructor(private val awsSdk: ObjectNode?) {
         get() =
             awsSdk?.getStringMember("endpointsConfigPath")?.orNull()?.value?.let { Paths.get(it) }
 
+    /** Path to the `default-partitions.json` configuration */
+    val partitionsDotJson: Path?
```

Comment:
For consistency with the other file path fields, this should be named `partitionsConfigPath`.

Reply:
renamed both